peer/brontide.go

```diff
@@ -4111,6 +4118,29 @@ func (p *Brontide) handleRemovePendingChannel(req *newChannelMsg) {
 	p.addedChannels.Delete(chanID)
 }
 
+// handlePeerStorageMessage handles `PeerStorage` message, it stores the message
+// and sends it back to the peer as an ack.
+func (p *Brontide) handlePeerStorageMessage(msg *lnwire.PeerStorage) error {
+	// Check if we have the feature to store peer backup enabled.
+	if p.cfg.PeerDataStore != nil {
+		// Store the data.
+		err := p.cfg.PeerDataStore.Store(msg.Blob)
+		if err != nil {
+			return err
+		}
+
```

Comment:
think the logic of checking if the peer at least has a channel with us before storing is missing.

Reply:
I think it does not need to have a channel to store with us to store its data with us according to the spec